awx/main/tests/functional/api/test_notifications.py

```diff
@@ -0,0 +1,24 @@
+import pytest
```

Comment:
Please note that I will be adding more tests to this file, it's just a placeholder for now (adding things piecemeal as they pass)!